setup.py

```diff
@@ -35,6 +45,7 @@ def run_tests(self):
 
 setup_requirements = [
     "cython>=0.25.2",
+    "numpy>=1.11.3",
 ]
 
 install_requirements = [
```

Comment:
Should have added `numpy` to the `install_requirements` as well, but missed that for some reason. Adding it now in PR ( https://github.com/jakirkham/cyminmax/pull/36 ).